doc/howto/install-dune.rst

```diff
@@ -0,0 +1,39 @@
+How to Install Dune
+===================
+
+Dune is available as an Opam package. First, make sure that Opam is installed:
+
+  .. code:: console
+
+   $ opam --version
+   2.1.5
+
+Any version greater than 2.0.0 is supported, though at least 2.1.0 is preferred.
```

Comment:
To me these read a bit better. However, newer vs greater in versions is a bit of a grey zone, since "newer" reads better but "greater" is technically more accurate (2.0.45 might be newer than 2.1.0 after all), but greater sounds a bit like a value assesment. @christinerose what would you say, I'm honestly curious!

```suggestion
Any version newer than 2.0.0 is supported, though preferably at least 2.1.0.
```

or 

```suggestion
Any version newer than 2.0.0 is supported, though for the best experience at least 2.1.0 is recommended.
```

Reply:
"higher"?

Reply:
Yes, that's perfect!